### ISSUE TITLE
Movie accepts -N twice

### ISSUE THREAD

**Issue by Esteban82** [GenericMappingTools/gmt#5604]
I run the following command with the -N twice and it worked. The last -N was used. 

It is not really a bug because it worked, but maybe it's not a good idea to let this happen.

`gmt movie "Master_Hemisferico.sh" -N"TierraGirando0" -T"temp_col0.txt" -C15cx15cx100  -Fmp4 -Gblack -Nblack2`

**Comment by PaulWessel** [comment]
Agree, but this is a general problem for most non-common options.  Most module-level options do not check if they have already been set, so the last settings are used and there is no warning.  To prevent this everywhere we would need to do something like this (here for an imaginary option **-Q**):

```
case 'Q':
    n_errors += gmt_report_duplicate_option (API, opt->option, Ctrl->Q.active);
    Ctrl->Q.active = true;
    /* Parsing -Q now */
    ....
break;
```

where _gmt_report_duplicate_option_ would check if the 3rd argument is already true and print an error like (Option Q: Already set!" and return 1, else 0.
Also, there are a few options across GMT that are repeatable, so they should _not_ have this check.  So if we were to do this we would

1. Need a clever script (python, @meghanrjones or @seisman?) or short C program that could read just the parse () function in a module,  find the case statements and insert the _gmt_report_duplicate_option_ call with the correct letter for the "active" variable.
2. Manually check each revised module and make sure this worked and remove any calls for repeatable options.

Perhaps a safer and more benign _gmt_report_duplicate_option_ function would just issue a warning and say previous settings are overwritten, but not return an error code.

**Comment by meghanrjones** [comment]
Closed by #5608
